site/README.md

```diff
@@ -125,7 +125,7 @@ $ make protobuf_doc_gen
 
 ### REST API docs
 
-The REST API docs are built using a JSON definition generated by Maven's Swagger plugin. This definition is automatically generated when Maven builds Pulsar. In the rare case when you do need to re-generate this JSON definition, run `mvn package` to build all of Pulsar. Then you'll need to copy the new definition into the `site/_data` folder using this command:
+The REST API docs are built using a JSON definition generated by Maven's Swagger plugin. This definition is automatically generated when Maven builds Pulsar. In the rare case when you do need to re-generate this JSON definition, run `mvn package -P swagger` to build all of Pulsar. Then you'll need to copy the new definition into the `site/_data` folder using this command:
```

Comment:
The `admin-rest-api-swagger.json` should already be generated automatically during the website build. (I think we should remove from git)

I think the issue is that the website build is not passing the ` -P swagger`. I'll fix it now.